extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking/index.js

```diff
@@ -155,7 +172,7 @@ function PanelMeasurementTableTracking({ servicesManager, extensionManager }) {
           title="Measurements"
           amount={displayMeasurements.length}
           data={displayMeasurements}
-          onClick={() => {}}
+          onClick={() => { }}
```

Comment:
Someone's trying to get their "number of additions" up 👀

Reply:
I think we don't have linting on commit, but we do have it on save.